package.json

```diff
@@ -3,6 +3,9 @@
   "version": "2.2.3",
   "description": "A parser for the SPARQL query language",
   "author": "Ruben Verborgh <ruben.verborgh@gmail.com>",
+  "dependencies": {
+    "n3": "^1.0.4"
```

Comment:
Why the need for an N3/Turtle parser? Can this be added as a devDependency?

Reply:
Good question. @RubenVerborgh maybe https://github.com/rdfjs/data-model would suffice?

Reply:
Not `devDependency`; it's the default factory.

Reply:
(We can remove this later; main point is to get to RDF/JS now. Feel free to add an issue.)